### PR TITLE
Updated instructions for all modules

### PR DESCRIPTION
**Pull request by LaurieHeer**
All modules had at least minor changes to instructions. Steps were added to create data to replace missing sample data.

### CODE REVIEW

File: Instructions/Labs/LAB[MB-910]_M01_Lab3_Manage_Customers_and_Activities.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ Module 1: Learn the Fundamentals of Dynamics 365 Marketing
...
+Defining and working with account and contact records in Dynamics 365 applications is one of the most common activities you will do. Once accounts and contacts are created, activities such as phone calls, tasks, and appointments will represent the interactions that you have with your customers.
 
 ## Lab Setup
 
   - **Estimated Time**: 15 minutes
 
 ## Instructions
 
-In this exercise, you will be working with common records that are leveraged by all the first party customer engagement apps. 
+In this exercise, you will be working with common records that are used by all Dynamics 365 for Customer Engagement apps.
```
Comment (CeleneA) [inline Instructions/Labs/LAB[MB-910]_M01_Lab3_Manage_Customers_and_Activities.md:22]: There is no product called Dynamics 365 for Customer Engagement.  please keep "first party customer engagement apps"

File: Instructions/Labs/LAB[MB-910]_M01_Lab1_Validate lab environment.md [hunk 2]
```diff
@@ -30,14 +30,14 @@ Exercise 1 - Access the Dynamics 365 application
 
 **Important:** Please be aware that this tenant and the Dynamics 365 organization will be shared with the other students in your classroom, like employees would share a tenant when using the Dynamics 365 instance belonging to their organization. Do not use any PII (personally identifiable information) when creating records. It is also good practice to use your username prefix (ex., **mollyc**) in front of all records, data, apps, flows, etc. you create.
 
-3. Feel free to explore the Power Platform admin center but **do not make any changes.**
+3. Feel free to explore the Power Platform admin center but **do not make any changes**.
 
 ### Task 2 – Access the Dynamics 365 application
 
-1.  Expand the grid button at the top left of the screen, directly to the left of **Power Platform admin center.** Select **Dynamics 365.**
+1.  Expand the App Launcher domino button at the top left of the screen, directly to the left of **Power Platform admin center. Click on the Dynamics 365 **Marketing** app to launch it.
```
Comment (CeleneA) [inline Instructions/Labs/LAB[MB-910]_M01_Lab1_Validate lab environment.md:38]: please change all instances of "click on...." to select (accessibility consideration)

**Review by CeleneA — APPROVED** [review]

File: Instructions/Labs/LAB[MB-910]_M01_Lab5_Create_a_Segment.md [hunk 3]
```diff
@@ -29,20 +30,20 @@ During this exercise you will see that it is very easy to create customer segmen
 
 5. Select **Add query block** to create a query against the contact table. 
 
-6. Select the ghost text **Select attribute**. 
+6. Click the **Select attribute** control.
```
Comment (CeleneA) [inline Instructions/Labs/LAB[MB-910]_M01_Lab5_Create_a_Segment.md:34]: change "click" to "select"

File: Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md [hunk 4]
```diff
@@ -75,13 +77,13 @@ Upon completion of the lab, you will have completed the following:
 
 6. In the **Name** field enter **Home Security – Your Initials**. 
 
-7. In the **Subject** field enter the text **Keep you family Safe**!
+7. In the **Subject** field enter the text **Keep you family Safe**! You may need to scroll to the top of the Design page to reveal the Subject field.
 
 8. In the message window, locate and select the small image **above** the text a **Short headline goes here**. 
 
-9. Hover over and select the **Image gallery** button. 
+9. In the Propterties Pane, hover over and select the **Image gallery** button.
```
Comment (CeleneA) [inline Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md:85]: typo

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

File: Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md [hunk 5]
```diff
@@ -107,15 +109,15 @@ Upon completion of the lab, you will have completed the following:
...
 22. Click to select the entire section below the section you just modified. Press the **trash can** icon to remove the section. 
 
-23. Repeat the process for the remaining sections in the message. 
+23. Repeat the process for the remaining sections or other elements in the message.
 
 24. Before we make the email live, we will check it for errors. On the **Command Bar**, select the **Check for Errors** button. 
 
 25. Once you have verified that there are no errors, on the **Command Bar**, select the **Go Live** button. 
 
 26. Click the drop-down arrow next to the **Save** button on the command bar and select **Save and Close**.
```
Comment (CeleneA) [inline Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md:119]: change to "select" rather than "click"

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

**Review by CeleneA — APPROVED** [review]

File: Instructions/Labs/LAB[MB-910]_M01_Lab3_Manage_Customers_and_Activities.md (hunk 1 above)
Comment (LaurieHeer) [reply Instructions/Labs/LAB[MB-910]_M01_Lab3_Manage_Customers_and_Activities.md:22]: Thank you for pointing out that the CE license is no longer being offered and the correct syntax is to make customer engagement lowercase for the online product. This is a fundamentals class, so technical jargon should be reduced unless it is adding value.

File: Instructions/Labs/LAB[MB-910]_M01_Lab1_Validate lab environment.md (hunk 2 above)
Comment (LaurieHeer) [reply Instructions/Labs/LAB[MB-910]_M01_Lab1_Validate lab environment.md:38]: Updated

File: Instructions/Labs/LAB[MB-910]_M01_Lab5_Create_a_Segment.md (hunk 3 above)
Comment (LaurieHeer) [reply Instructions/Labs/LAB[MB-910]_M01_Lab5_Create_a_Segment.md:34]: Updated

File: Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md (hunk 4 above)
Comment (LaurieHeer) [reply Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md:85]: Updated

File: Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md (hunk 5 above)
Comment (LaurieHeer) [reply Instructions/Labs/LAB[MB-910]_M01_Lab7_Dynamics_365_Marketing_capstone_lab.md:119]: Updated

**Comment by CeleneA** [comment]
Approved

**Comment by hope77777** [comment]
Cam this be merged?

**Comment by LaurieHeer** [comment]
Yes
On 5/17/2021 1:34:17 PM, Hans Hope - LODS ***@***.***> wrote:
Cam this be merged?

**Comment by hope77777** [comment]
Looks like the pull was closed before it was merged.

**Pull request closed without merging**